contrib_bots/bots/foursquare/foursquare.py

```diff
@@ -82,7 +89,7 @@ def send_info(self, message, letter, client):
                 content=letter,
             ))
 
-    def handle_message(self, message, client, state_handler):
+    def handle_message(self, message, query, client, state_handler):
```

Comment:
I've noticed two approaches to  `client.send_message` in the bots:
- putting it in`handle_message`
- putting it in `send_info` 
Since we're striving for consistency, we might want to adjust the code in all the bots to follow the same pattern. Which one of those two would you think is better?

Also, since:
```
         if message['type'] == 'private':
             client.send_message(dict(
                 type='private',
                 to=message['sender_email'],
                 content=letter,
             ))
         else:
             client.send_message(dict(
                 type='stream',
                 subject=message['subject'],
                 to=message['display_recipient'],
                  content=letter,
              ))
```
repeats quite a lot across many bots, we might consider making a shared function - what do you think about that?